mmdet/apis/ote/apis/detection/nncf_task.py

```diff
@@ -65,6 +66,7 @@ def __init__(self, task_environment: TaskEnvironment):
         self._val_dataloader = None
         self._compression_ctrl = None
         self._nncf_preset = "nncf_quantization"
+        self._optimization_type = ModelOptimizationType.NNCF
         check_nncf_is_enabled()
         super().__init__(task_environment)
```

Comment:
Is it Ok to call parent class constructor after setting custom optimization type here?

Reply:
ofc no, thanks